crystal/core.cr

```diff
@@ -1,456 +1,454 @@
 require "time"
 
+require "readline"
 require "./types"
 require "./error"
 require "./printer"
 require "./reader"
-require "./readline"
 
 module Mal
-
-macro calc_op(op)
+  macro calc_op(op)
```

Comment:
Is this formatting correct? Indentation looks weird. I think other `macro ... end` blocks are also the case.

Reply:
Looks like the formatter doesn't touch macros, fixed!